Bottom: 50d55f1fdc584df3e602f1e3f4e30968bcfb1b6d
Top:    fbfaac9989bcfefc7d614285fb1b243bcc229efc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-09 17:30:24 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b2539ca..7cdb152 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2686,7 +2686,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
